src/test/ui/typeck/issue-93486.stderr

```diff
@@ -5,11 +5,6 @@ LL |         vec![].last_mut().unwrap() = 3_u8;
    |         -------------------------- ^
    |         |
    |         cannot assign to this expression
-   |
-help: consider dereferencing here to assign to the mutable borrowed piece of memory
-   |
-LL |         *vec![].last_mut().unwrap() = 3_u8;
-   |         +
```

Comment:
Do we know why we lost this message? (It seems like it is correct unlike the one in the previous commit so I'm surprised it's missing.)